charts/pega/README.md

```diff
@@ -968,7 +971,11 @@ tls:
    port: 443
    targetPort: 8443
    # set the value of CA certificate here in case of baremetal/openshift deployments - CA certificate should be in base64 format
-   cacertificate: 
+   # pass the certificateChainFile file if you are using certificateFile and certificateKeyFile
+   cacertificate:
+   # SSL certificate and private key as a PEM format
```

Comment:
provide the SSL certificate and private key as a PEM format

Reply:
done